src/lib.rs

```diff
@@ -21,7 +21,11 @@
 //! assert_eq!(format!("{}", Currency::EUR.symbol()), "€");
 //! ```
 
-#[derive(PartialEq)]
+#[cfg(feature = "serde")]
```

Comment:
Just need to change to the new feature name.

Reply:
oops